test/eChronicler/e_chronicler/journal_entry_test.exs

```diff
@@ -0,0 +1,18 @@
+defmodule EChronicler.EChronicler.JournalEntryTest  do
+  use EChronicler.DataCase, async: true
+  import Ecto.Query, warn: false
+
+  alias EChronicler.EChronicler.JournalEntries
```

Comment:
Github is showing this helpful message here on my end.  Good idea to remove things that aren't being used.  It's basically just saying "Using `alias`, you import the functionality from this module but dont use it. Why not delete it?"

![image](https://user-images.githubusercontent.com/25881824/146426439-6cc238ed-917b-4ea0-9289-625738438af9.png)